client/me/purchases/manage-purchase/notices.tsx

```diff
@@ -1063,7 +1066,23 @@ class PurchaseNotice extends Component<
 	renderTrialNotice( productSlug: string ) {
 		const { moment, purchase, selectedSite, translate } = this.props;
 		const onClick = () => {
-			return page( `/plans/${ selectedSite?.slug }` );
+			const upgradePlanSlug =
+				productSlug === PLAN_ECOMMERCE_TRIAL_MONTHLY
+					? getPlan( PLAN_ECOMMERCE )?.getStoreSlug()
+					: getPlan( PLAN_BUSINESS )?.getStoreSlug();
+
+			this.props.recordTracksEvent( 'calypso_business_expired_trial_upgrade_cta_clicked', {
```

Comment:
I think this event name would be more in keeping with naming convention. The other events on this view are `calypso_subscription_*`

```suggestion
			this.props.recordTracksEvent( 'calypso_subscription_trial_notice_cta_clicked', {
```